easybuild/easyblocks/g/gromacs.py

```diff
@@ -208,6 +209,19 @@ def configure_step(self):
             # version of GROMACS. Just prepare first part of cmd here
             plumed_cmd = "plumed-patch -p -e %s" % engine
 
+        # Ensure that the GROMACS log files report how the code was patched
+        # during the build, so that any problems are easier to diagnose.
+        if '-DGMX_VERSION_STRING_OF_FORK=' not in self.cfg['configopts']:
+            if plumed_root:
+                plumed_version = get_software_version('PLUMED')
+                if plumed_version:
+                    gromacs_version_string_suffix = '%s-%s' % (plumed_version, VERBOSE_VERSION)
```

Comment:
@mabraham note that plumed_version does not contain the text "PLUMED" only the version, also if plumed_root is set plumed_version will always return something, so this could be reduced to just
```
plumed_version = get_software_version('PLUMED')
gromacs_version_string_suffix = 'PLUMED-%s-%s' % (plumed_version, VERBOSE_VERSION)
```
skipping the else paert of the "if plumed_version"

Reply:
Thanks, done!